src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -19,17 +19,18 @@ Cypress.Commands.add('verifyOptions', () => {
   cy.get('#facility-type-dropdown').select(
     'Community providers (in VA’s network)',
   );
-  cy.get('#service-type-dropdown').should('not.have.attr', 'disabled');
+  cy.get('#service-typeahead').should('not.have.attr', 'disabled');
 
   // CCP pharmacies dont have services available
   cy.get('#facility-type-dropdown').select(
     'Community pharmacies (in VA’s network)',
   );
-  cy.get('#service-type-dropdown').should('not.have', 'disabled');
+  cy.get('#service-typeahead').should('not.have', 'disabled');
 });
 
-describe.skip('Facility search', () => {
+describe('Facility search', () => {
   before(() => {
+    if (!Cypress.env('CIRCLECI')) this.skip();
```

Comment:
If `CIRCLECI` is not set in `cypress.json` environment variables then the test bombs with:

>  TypeError: Cannot read property 'skip' of undefined

I take it that Jenkins/CircleCI are setting this in our CI environments, but do we have guidance for users executing these e2e suite locally around setting the `CIRCLECI` environment variable?

Reply:
Thanks for testing it and good catch. This is actually just a minor syntax issue in the `before()` hook; the `CIRCLECI` variable doesn't have to be set. I just pushed up a fix.

Reply:
Verified -- test skips successfully now when `CIRCLECI` is unset.